packages/react-ui/src/theme/carto-theme.d.ts

```diff
@@ -9,20 +9,44 @@ import { Palette, PaletteColor } from '@mui/material';
 export const cartoThemeOptions: ThemeOptions;
 
 type Modify<T, R> = Omit<T, keyof R> & R;
+
 type CustomPaletteColor = Modify<
   PaletteColor,
   {
     relatedDark: string;
     relatedLight: string;
     background: string;
     outlinedBorder: string;
-    hint: string;
+  }
+>;
+type CustomBrandPaletteColor = Modify<
```

Comment:
👍🏻 Yes, I think that makes more sense... don't you?

Reply:
Yep, my bad for trying to go faster. Next time I'll do it this way.